exercise3/src/camera_demo/src/actuation_node.cpp

```diff
@@ -0,0 +1,93 @@
+// Listen to the /stop topic and stop the robot when a message is received
+
+#include <chrono>
+#include <memory>
+#include <string>
+#include <vector>
+#include <list>
+#include <thread>
+#include <cactus_rt/tracing.h>
+
+#include "rclcpp/experimental/fifo_sched.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/empty.hpp"
+
+using namespace std::chrono_literals;
+
+using FileSink = cactus_rt::tracing::FileSink;
+using TraceAggregator = cactus_rt::tracing::TraceAggregator;
+using ThreadTracer = cactus_rt::tracing::ThreadTracer;
+
+
+class ActuationNode : public rclcpp::Node
+{
+public:
+  ActuationNode()
+  : Node("actuation_node")
+  {
+    StartTracing("actuation_node", "actuation_node.json");
```

Comment:
This code should be in `int main`, as there should be one per process. I know that you have only one node per process, but we should consider merging them all so the trace file comes out with all threads in a single file.

Also it's not a JSON file but a proto file. The extension we should use is `.perfetto` for this particular case. Also, to make it easier for people to identify which trace file is from which exercise, we should name the trace file `exercise3.perfetto` (or maybe `exercise3a.perfetto`).